src/pages/FailedReason/index.tsx

```diff
@@ -23,6 +23,8 @@ import {
   tBase,
 } from '@/constants/constants';
 
+const orderId = 4;
```

Comment:
I dont like that you hardrcode ID think about other option or write explanation

Reply:
I just didn't see that the page from which the transition is made is ready, I fixed it